UnitTesting/SpecificLeaksEventsTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using WaterLog_Backend;
+using WaterLog_Backend.Controllers;
+using WaterLog_Backend.Models;
+using WebApplication1;
+
+namespace UnitTesting
+{
+    class SpecificLeaksEventsTest
+    {
+       
+        private IQueryable<SegmentLeaksEntry> mockData = new List<SegmentLeaksEntry>
```

Comment:
I see you are creating an IQueryable here, what is the reason for this? Why not just create a list?

Reply:
I initially had tests that made use of IQueryable  methods, planning on working on them the next sprint.

Reply:
Okay cool, hoping to see a 📦 change in the unit tests :)